src/components/FlawHistory.vue

```diff
@@ -62,4 +82,51 @@ function isDateField(field: string) {
       </label>
     </div>
   </LabelCollapsible>
+  <div v-if="history" class="pagination-controls gap-1 my-2">
+    <button
+      type="button"
+      tabindex="-1"
+      class="btn btn-sm btn-secondary rounded-end-0"
+      :disabled="currentPage === 1"
+      @click="changePage(currentPage - 1)"
+    >
+      <i class="bi bi-arrow-left fs-5" />
+    </button>
+    <button
+      v-for="page in pages"
+      :key="page"
+      tabindex="-1"
+      class="osim-page-btn btn btn-sm rounded-0 btn-secondary"
+      style="width: 34.8px;"
+      :disabled="page === currentPage || page === '..'"
+      @click.prevent="changePage(page as number)"
+    >
+      {{ page }}
+    </button>
+    <button
+      type="button"
+      tabindex="-1"
+      class="btn btn-sm btn-secondary rounded-start-0"
+      :disabled="currentPage === totalPages || totalPages === 0"
+      @click.prevent="changePage(currentPage + 1)"
+    >
+      <i class="bi bi-arrow-right fs-5" />
+    </button>
+  </div>
```

Comment:
We have this exact markup each time we use the pagination, maybe is worth creating its own component that receive the required data from props?
https://github.com/RedHatProductSecurity/osim/blob/48740f7834538a7ce11e44762d4966b5a878300e/src/components/FlawAffects/FlawAffects.vue#L372-L402
https://github.com/RedHatProductSecurity/osim/blob/48740f7834538a7ce11e44762d4966b5a878300e/src/components/FlawTrackers.vue#L118-L144